core/src/Cuda/Kokkos_Cuda_Parallel.hpp

```diff
@@ -70,6 +70,15 @@
 #include <KokkosExp_MDRangePolicy.hpp>
 #include <impl/KokkosExp_IterateTileGPU.hpp>
 
+#include <thrust/host_vector.h>
+#include <thrust/device_vector.h>
+#include <thrust/generate.h>
+#include <thrust/sort.h>
+#include <thrust/copy.h>
+#include <thrust/iterator/counting_iterator.h>
+
+#define KOKKOS_ENABLE_THRUST
```

Comment:
Eventually this should be a KOKKOS_ENABLE_IMPL_CUDA_THRUST option which is available through CMake, but for now we need the testing, so lets do that last.